design/ceph/object/ceph-object-realm-pull.md

```diff
@@ -0,0 +1,88 @@
+# Rook Ceph Object Realm Pull
+
+## Prerequisites
+
+A Rook Ceph cluster.
+
+## Ceph Object Realm Pull Walkthrough
+
+The resource descibed in this design document represents pull a realm in the [Ceph Multisite data model](/design/ceph/object/ceph-multisite-overview.md).
+
+
+### Pulling the Realm from another Ceph Cluster
+
+#### Config
+
+When the storage admin is ready to sync data from a Ceph cluster with multisite set up (primary cluster) to another Rook Ceph cluster (pulling cluster), the pulling cluster needs to have information about the primary cluster's realm and zone group.
+
+To do this, the storage admin needs to create a ceph-object-realm-pull resource on the pulling cluster with the same name as the realm from the primary cluster.
+
+The endpoint in the `pullEndpoint` section is an endpoint of an object-store in the master zone of the realm. 
+
+This endpoint must be resolvable from the pulling cluster.
+
+This example `ceph-object-realm-pull.yaml`, pulls `my-realm` from a primary Ceph cluster with endpoint http://my-realm-zone-b-objectstore-1:80 in its master zone.
+```yaml
+apiVersion: ceph.rook.io/v1alpha1
+kind: CephObjectRealmPull
+metadata:
+  name: my-realm
+  namespace: rook-ceph-2
+spec:
+  pull:
+    endpoint: http://my-realm-zone-b-objectstore-1:80
+```
+
+Now create the ceph-object-realm-pull
+```bash
+kubectl create -f ceph-object-realm-pull.yaml
+```
+
+Once the creation of the ceph-object-realm-pull resource has been kicked off, the realm will be pulled on the pulling cluster. 
+
+If this realm pull succeeds, ceph-object-realm and ceph-object-zone-group resources will automatically be created. 
+
+The ceph-object-realm and ceph-object-zone-group resource on the pulling Ceph cluster will have the same name and config as the realm and zone group on the primary cluster.
+
+These resources just ensure the pulling cluster has the same ceph-object-realm and ceph-object-zone-group resources as the primary cluster.
+
+When these resources are created, neither the realm nor zone group is being recreated. 
+
+After these resources are automatically created the ceph-object-realm-pull resource updated as Complete.
+
+#### Steps
+1. At this point the Rook operator recognizes that a new ceph-object-realm-pull resource needs to be configured. The operator will and will pull the realm.
+2. If the realm pull succeeds, a ceph-object-realm and ceph-object-zone-group resource will be created and the ceph-object-realm-pull resource is updated as complete. 
```

Comment:
The realm and zone groups created are just at the Ceph level, right? Generating CRs for those doesn't seem necessary assuming the operator has no more action to take. The realm pull already takes care of everything that is needed internally to Ceph, right? CRs should only be needed if there is desired state the operator needs to manage.